curation.yml

```
@@ -664,3 +664,68 @@
         작가: 김인호
         설명: 🎯 코딩 테스트 준비의 핵심을 한 눈에! 알고리즘 문제 유형 파악과 효율적인 학습 전략으로 합격의 지름길을 안내하는 실전 가이드북
 
+  11th:
+    Frontend:
+      - 제목: 어떻게 서버에 요청을 최적으로 보낼까? 배치 작업부터 제너레이터까지
+        주소: https://witch.work/ko/posts/blog-api-call-optimization
+        작가: 김성현a
+        설명: 🤹‍♀️ 서버 요청의 효율성과 안정성을 높이기 위해, 배치 처리와 작업 큐, 제너레이터를 활용한 최적의 요청 관리 방법을 탐구하는 실용적인 가이드입니다.
+      - 제목: React 컴포넌트 Unmount시에 CSS 애니메이션 적용하기
+        주소: https://1lsang.vercel.app/posts/presence
+        작가: 박일상
+        설명: 🎬 React 컴포넌트의 Unmount시 애니메이션 처리가 어려운 이유와 해결방법을 상세히 설명한 글로, Radix UI의 Presence 컴포넌트 분석을 통해 애니메이션 이벤트와 내부 상태 기반의 언마운트 제어로 자연스러운 Exit 애니메이션을 구현하는 방법을 배울 수 있습니다.
+      - 제목: CSS interpolate-size 기반 아코디언,  JS보다 빠를까? 실측 성능 테스트
+        주소: https://mong-blog.tistory.com/entry/CSS-interpolate-size-%EA%B8%B0%EB%B0%98-%EC%95%84%EC%BD%94%EB%94%94%EC%96%B8-JS%EB%B3%B4%EB%8B%A4-%EB%B9%A0%EB%A5%BC%EA%B9%8C-%EC%8B%A4%EC%B8%A1-%EC%84%B1%EB%8A%A5-%ED%85%8C%EC%8A%A4%ED%8A%B8
+        작가: 금정민
+        설명: 🧪 CSS interpolate-size 기반 아코디언이 JS 방식보다 CPU 부하가 적고 렌더링 최적화에 효율적이라는 것을 실측 성능 테스트를 통해 입증한 유익한 글입니다.
+      - 제목: 피그마 무료 플랜에서 컬러 코드 추출 자동화하기! (feat. Figmable CLI 배포)
+        주소: https://velog.io/@byungsker/Figmable-CLI-%EB%B0%B0%ED%8F%AC%EB%AC%B4%EB%A3%8C%ED%94%8C%EB%9E%9C%EC%97%90%EC%84%9C-%ED%94%BC%EA%B7%B8%EB%A7%88-rest-api%EB%A1%9C-%ED%86%A0%ED%81%B0-%EA%B0%80%EC%A0%B8%EC%98%A4%EA%B8%B0#51-varibaleslocal-%EC%82%AC%EC%9A%A9-%EB%B6%88%EA%B0%80
+        작가: 이병우
+        설명: 🎨 피그마 무료 플랜에서 컬러 코드를 자동으로 추출하고 CSS에 쉽게 적용할 수 있는 Figmable CLI로 디자인 시스템 동기화의 번거로움을 해결하세요!
+
+    Backend:
+      - 제목: Spring Boot JAR에서 진짜 main()은 따로 있다?
+        주소: https://velog.io/@byeolhaha/Jar%EC%9D%98-Main-method%EA%B0%80-%EB%82%B4%EA%B0%80-%EB%A7%8C%EB%93%A0-Main-method%EA%B0%80-%EC%95%84%EB%8B%88%EB%9D%BC%EA%B3%A0
+        작가: 김별
+        설명: 🚀 Spring Boot JAR의 실행 구조와 Nested JAR 방식을 이해하고 싶다면, Java의 기본 ClassLoader 한계부터 Spring Boot의 해결책까지 상세히 설명된 이 글을 통해 JAR 파일의 내부 동작 원리를 파악할 수 있습니다.
+      - 제목: MySQL 데드락 발생 프로시저 개선기 | 내가 그린
```

```suggestion
      - 제목: MySQL 데드락 발생 프로시저 개선기
```

글 제목에서 블로그 이름은 일반적으로 지우는 것 같아요